Actions/CreateApp/action.yaml

```diff
@@ -35,7 +39,7 @@ inputs:
 runs:
   using: composite
   steps:
-    - run: ${{ github.action_path }}/CreateApp.ps1 -actor '${{ inputs.actor }}' -token '${{ inputs.token }}' -project '${{ inputs.project }}' -type '${{ inputs.type }}' -name '${{ inputs.name }}' -publisher '${{ inputs.publisher }}' -idrange '${{ inputs.idrange }}' -directCommit ('${{ inputs.directCommit }}' -eq 'Y')
+    - run: ${{ github.action_path }}/CreateApp.ps1 -actor '${{ inputs.actor }}' -token '${{ inputs.token }}' -project '${{ inputs.project }}' -type '${{ inputs.type }}' -name '${{ inputs.name }}' -publisher '${{ inputs.publisher }}' -idrange '${{ inputs.idrange }}' -directCommit ('${{ inputs.directCommit }}' -eq 'Y') -parentTelemetryScope '${{ inputs.parentTelemetryScope}}' -telemetryEventId 'DO0072'
```

Comment:
The order of these parameters in the run line should match the order of parameters in the PowerShell script and the order in the yaml file - else tests will fail.